toqito/state_opt/ppt_distinguishability.py

```diff
@@ -91,188 +94,116 @@ def ppt_distinguishability(
         :filter: docname in docnames
 
 
-    :param states: A list of states provided as either matrices or vectors.
-    :param dist_method: The method of distinguishing states.
+    :param vectors: A list of states provided as either matrices or vectors.
     :param probs: Respective list of probabilities each state is selected.
-    :param dist_method: Method of distinguishing to use.
-    :param strategy: Returns strategy if :code:`True` and does not otherwise.
+    :param subsystems: A list of integers that correspond to the complex Euclidean space dimensions.
+    :param dimensions: A list of integers that correspond to the dimensions of the subsystems.
+    :param strategy: The method of distinguishing states.
+    :param solver: The SDP solver to use.
+    :param primal_dual: Option for the optimization problem.
     :return: The optimal probability with which the states can be distinguished
              via PPT measurements.
 
     """
-    __is_states_valid(states)
-    if probs is None:
-        probs = [1 / len(states)] * len(states)
-    __is_probs_valid(probs)
-
-    _, dim_y = states[0].shape
-
-    # The variable `states` is provided as a list of vectors. Transform them
-    # into density matrices.
-    if dim_y == 1:
-        for i, state_ket in enumerate(states):
-            states[i] = state_ket * state_ket.conj().T
-
-    if strategy:
-        return primal_problem(states, probs, dist_method)
-    return dual_problem(states, probs, dist_method)
-
-
```

Comment:
![image](https://github.com/vprusso/toqito/assets/66048318/8018f814-4dfa-401e-a757-1c8871e0e8bd)

Seems like switching to `picos` worked! `pytest` no longer flags a line in this function to not be covered by tests.

Reply:
Hurray!